plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -119,6 +120,14 @@ public void testReadFromView()
         }
     }
 
+    @Test
+    @Override
+    public void testSpecialValueOnApproximateNumericColumn()
+    {
+        assertThatThrownBy(super::testSpecialValueOnApproximateNumericColumn)
+                .hasMessageContaining("Failed to insert data: The incoming tabular data stream (TDS) remote procedure call (RPC) protocol stream is incorrect.");
```

Comment:
looks wrong. why is it OK to skip this test for SQL Server?

Reply:
the test probably should be controlled by testing behavior ("can store NaN and infinity?")
in case of No, it should verify that every insert fails (NaN, +inf and -inf separately)
in case of Yes, it should verify correctness on subsequent select queries

no test overrides should be needed